### PR TITLE
Missing tests for existing breakdown row table functionality

### PR DESCRIPTION
**Pull request by drewmiley**
## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [x] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.

### CODE REVIEW

File: app/views/confirmation/confirmation_new.scala.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
 @import forms2._
```
Comment (JGRobinson) [inline app/views/confirmation/confirmation_new.scala.html]: Do we need to do this refactor now ?

Comment (JGRobinson) [inline app/views/confirmation/confirmation_new.scala.html]: Do we need to do this refactor now.

**Comment by drewmiley** [comment]
@JGRobinson 👍 This should form part of the work for AMLS-3402/AMLS-4218 due to refactor of the breakdown display.

**Comment by ajchilts** [comment]
I am going to close this PR as the code base has moved on a great deal since it was raised. We will however; keep the branch should we need to come back to this. 

As this is the fee breakdown table only I am reluctant to spend too much time on this. We can address test coverage of the table at some point in the future when we have less pressing issues.

**Pull request closed without merging**